dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/relationship/HibernateRelationshipStore.java

```diff
@@ -88,6 +90,49 @@ public HibernateRelationshipStore(
     super(entityManager, jdbcTemplate, publisher, Relationship.class, aclService, true);
   }
 
+  @Override
```

Comment:
please remind me again 😅 what is the reason this store implements an interface?

Reply:
At the moment is still used in services/stores of other entities.
Maybe in next PRs we will remove the interface as it will only be used internally by relationships.